docs/apache-airflow-providers-databricks/operators/sql.rst

```diff
@@ -84,3 +84,39 @@ An example usage of the DatabricksSqlOperator to perform statements from a file
     :language: python
     :start-after: [START howto_operator_databricks_sql_multiple_file]
     :end-before: [END howto_operator_databricks_sql_multiple_file]
+
+
+DatabricksSqlSensor
+===================
+
+Use the :class:`~airflow.providers.databricks.sensors.sql.DatabricksSqlSensor` to run the sensor
+for a specific Delta table on a Databricks SQL workspace.
```

Comment:
Did you mean on a Databricks SQL warehouse?

Reply:
Yes, changed.